src/garage/np/algos/cem.py

```diff
@@ -43,15 +47,19 @@ def __init__(self,
                  best_frac=0.05,
                  extra_std=1.,
                  extra_decay_time=100):
-        super().__init__(env_spec, policy, baseline, discount, max_path_length,
-                         n_samples)
-
-        self.n_samples = n_samples
         self.best_frac = best_frac
+        self.baseline = baseline
```

Comment:
to the extend possible (i know there is still some code which assumes public attrs), please prefix all of these with `_` to make them private (e.g. `self._baseline = baseline`)